pages/dkp/kommander/2.2/release-notes/2.2.0/index.md

```diff
@@ -355,7 +353,49 @@ Upgrading catalog applications using Spark Operator can fail when running `dkp u
 
 ### Minio Disk insufficient space when upgrading
 
-When upgrading DKP from v2.1.x to v2.2.x, the upgrade can fail due to insufficient space on the MinIO Disk. To avoid this issue, we recommend that you disable the `fluent-bit` Platform Application before upgrading. 
+When upgrading DKP from v2.1.x to v2.2.x, the upgrade can fail due to insufficient space on the MinIO Disk. To avoid this issue, we recommend that you disable the `fluent-bit` Platform Application before upgrading.
```

Comment:
```suggestion
When upgrading DKP from v2.1.x to v2.2.x, the upgrade can fail due to insufficient space on the MinIO Disk. To avoid this issue, D2iQ recommends that you disable the `fluent-bit` Platform Application before upgrading.
```